lib/livebook_web/controllers/session_controller.ex

```diff
@@ -173,6 +174,91 @@ defmodule LivebookWeb.SessionController do
     end
   end
 
+  def show_input_audio(conn, %{"token" => token}) do
```

Comment:
It would be great if we could have some controller tests aroundnthis, but I don't know how feasible it is. At least a basic one that checks the returned ranges and another request includes a range.